main.go

```diff
@@ -65,7 +65,16 @@ func main() {
 		os.Exit(1)
 	}
 
-	requestHandler := buildRequestHandler(watchdogConfig, watchdogConfig.PrefixLogs)
+	if watchdogConfig.ReadyEndpoint != "" {
```

Comment:
I think we can omit this because it will generate a lot of logs over the course of a year